### DISCUSSION TITLE
getThumbnail not returning webp format any more?

### DISCUSSION THREAD

**Discussion by mrpdean** [pimcore/pimcore#10430] [Q&A / Support] [answered]
On my Pimcore 6 site _getThumbnail_ in twig would always return an image in webp format if browsing in Chrome for example.

However on my Pimcore X site _getThumbnail_ is now always returning a JPEG.  I know I have webp set up correctly because running the _pimcore:thumbnails:image_ command generates the thumbnails in webp format.. however they do not get returned.

They do get return if I use the _pimcore_image_ editable, but not if I use _getThumbnail_ 

This returns webp images:

```
{{ pimcore_image("myImage", {
    "title": "Drag your image here",
    "width": 200,
    "height": 200,
    "thumbnail": "Hero"
}) }}
```

However, this only returns jpg images:

```
{% if editmode %}
    {{ pimcore_image("myImage",{"thumbnail": "Hero"}) }}
{% else %}
    {% set thumbnail = pimcore_image("myImage").getThumbnail("Hero") %}
    <img src="{{ thumbnail }}" width="{{ thumbnail.width }}" height="{{ thumbnail.height }}" />
{% endif %}
```

Has anyone else come across this issue?

I'm not sure if it's a bug or if something has changed in Pimcore X that I'm missing.

Thanks,

**Comment by brusch** [comment, marked as answer]
This is intentional, there's no more request-specific return format anymore, this is mainly about caching issues. 
Use `{{ thumbnail.html }}` instead, which returns a `<picture>` tag, so that the client can decide which format to use.

**Reply by mrpdean** [reply]
Thanks for clarifying.
What about if using the images in CSS background-image styles? Is that no longer possible for webp format?
